pyproject.toml

```diff
@@ -22,6 +22,7 @@ dependencies = [
     "semver",
     "setuptools",
     "typer",
+    "pyaml",
```

Comment:
py**y**aml

[pyaml](https://pypi.org/project/pyaml/) is something else - something that apparently produces invalid output as a "feature"
> One good example of such “caveat” is that e.g. {'foo': '123'} will serialize to foo: 123, which for PyYAML would be a bug, as 123 will then be read back as an integer from that, but here it’s a feature.

Reply:
And we see a case of potential typosquatting IRL :face_exhaling: